aws-s3/spring-cloud-starter-stream-source-s3/src/main/java/org/springframework/cloud/stream/app/s3/source/AmazonS3SourceConfiguration.java

```diff
@@ -57,7 +57,7 @@
 	private ResourceIdResolver resourceIdResolver;
 
 	@Bean
-	public TriggerProperties triggerProperties() {
+	public static TriggerProperties triggerProperties() {
```

Comment:
Apparently yes.  I'll merge by putting in inside `@EnableConfigurationProperties`, hopefully no lifecycle issues otherwise we will have to put inside another `@Configuration` class as that seems like the lesser of two evils... not a fan of static here.